ingestion/examples/sample_data/datasets/tables.json

```diff
@@ -678,7 +678,7 @@
           },
           {
               "id": "1cda9ecb-f4c6-4ed4-8506-abe965b64c87",
-              "name": "0005_dim_address_clean",
+              "name": "dim_address_clean",
```

Comment:
@harshach reverted it to 0005_dim_address_clean as it was giving Validation error in sample data for lifecycle